src/diffusers/loaders.py

```diff
@@ -109,6 +109,20 @@ def text_encoder_attn_modules(text_encoder):
     return attn_modules
 
 
+def text_encoder_aux_modules(text_encoder):
+    aux_modules = []
+
+    if isinstance(text_encoder, CLIPTextModel):
+        for i, layer in enumerate(text_encoder.text_model.encoder.layers):
+            mlp_mod = layer.mlp
+            name = f"text_model.encoder.layers.{i}.mlp"
+            aux_modules.append((name, mlp_mod))
+    else:
+        raise ValueError(f"do not know how to get aux modules for: {text_encoder.__class__.__name__}")
+
+    return aux_modules
```

Comment:
We can potentially increase the coverage of this function so that it support SDXL LoRA checkpoints by also adding `CLIPTextModelWithprojection`. Thoughts?

Reply:
Yep, let's do it in the subsequent PR (SDXL LoRAs are also the next thing I want to look at)